app/src/main/java/com/example/rwothoromo/developers/model/GithubUsersResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.example.rwothoromo.developers.model;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GithubUsersResponse {
+
+	@SerializedName("items")
+	private List<GithubUsers> githubUsers;
+
+	public GithubUsersResponse() {
+		githubUsers = new ArrayList<GithubUsers>();
+	}
+
+	public static GithubUsersResponse parseJSON(String response) {
```

Comment:
I am not sure this is needed either, but you don't necessarily have to remove it now until you have integrated the retrofit instance and you are sure you won't be needing it.

Reply:
I will be able to confirm once RetroFit is up and running.